_pages/robot/2015.html

```diff
@@ -4,6 +4,8 @@
 permalink: /robot/2015
 stylesheets:
 - banners
+- recycle
+- basic
```

Comment:
The name `basic` doesn't tell us what it actually is. That could mean a lot of things, couldn't it? Maybe try `fixedbackground` or something like that??